### ISSUE TITLE
Request: Plutus

### ISSUE THREAD

**Issue by CodeShakingSheep** [simple-icons/simple-icons#10988]
### Brand Name

Plutus

### Website

https://plutus.it/

### Popularity Metric

The Similarweb rank is 129,301. See https://www.similarweb.com/website/plutus.it

### Official Resources for Icon and Color

svg: https://plutus.it/images/logo-gold-d48ccf9d33.svg
color: `#d1ae52`

### Additional Comments

I will start working on this.

**Comment by uncenter** [comment]
https://plutus.it/terms-and-conditions states the following:

> 9.2 All intellectual property included in the Website or the Services, including, but not limited to, website logos, databases, website design, text and graphics, software, photos, videos, music, sounds and any combinations of the aforementioned files, and the intellectual property rights of software compilation associated source code and software (including small applications and scripts) belong to us and our licensors absolutely. You are not granted any licence or right to use these materials and other documents or items for any purpose other than in connection to your receipt of the Services. You may not copy, modify, transmit or use any of the foregoing materials or content for commercial purposes.

> 9.3 All rights contained in the name of the Website (including but not limited to business goodwill and trademarks, logos) belong to us or our licensor absolutely.

Please check the terms and conditions / legal documents before creating pull requests!

**Comment by CodeShakingSheep** [comment]
> https://plutus.it/terms-and-conditions states the following:
> 
> > 9.2 All intellectual property included in the Website or the Services, including, but not limited to, website logos, databases, website design, text and graphics, software, photos, videos, music, sounds and any combinations of the aforementioned files, and the intellectual property rights of software compilation associated source code and software (including small applications and scripts) belong to us and our licensors absolutely. You are not granted any licence or right to use these materials and other documents or items for any purpose other than in connection to your receipt of the Services. You may not copy, modify, transmit or use any of the foregoing materials or content for commercial purposes.
> 
> > 9.3 All rights contained in the name of the Website (including but not limited to business goodwill and trademarks, logos) belong to us or our licensor absolutely.
> 
> Please check the terms and conditions / legal documents before creating pull requests!

Yes, makes sense. Will do it in the future. Sorry for the overhead.

**Comment by uncenter** [comment]
No worries at all. I just found out that a few PRs I made recently were also https://github.com/simple-icons/simple-icons/labels/permission%20required so I've been extra cautious today (and will be going forward!).

**Comment by CodeShakingSheep** [comment]
Requested permission via their support form.

**Comment by PeterShaggyNoble** [comment]
See #13453
